### ISSUE TITLE
The MR is not as good as that in the paper.

### ISSUE THREAD

**Issue by Chen94yue** [liuwei16/ALFNet#2]
I have trained the proposed model on my server with P40 GPU.  The MR on Cityperson val dataset is 18% with one GPU (16% for two). Where might the problem arise?

**Comment by rintiunse** [comment]
i have the similar problem that using alfnet-2s, i spent 2s on per image  , so it's hard to acheive 20fps with gtx 1080ti. what's the problem ?

**Comment by VideoObjectSearch** [comment]
@Chen94yue Sorry for delayed reply.  Did you decrease the learning rate and load the trained weights to continue training?

**Comment by VideoObjectSearch** [comment]
@rintiunse  Sorry for delayed reply.  Did you make sure that your tensorflow is running on GPU?
I just re-run the exact experiments 4 times.  It approximately took 126~132 seconds on 500 images of 1024x2048 pixels on CityPersons validation set.
20fps is achieved on Caltech with images of 480x640 pixels as stated in the paper.

**Comment by Chen94yue** [comment]
> @Chen94yue Sorry for delayed reply. Did you decrease the learning rate and load the trained weights to continue training?

No, I didn't. Let me have a try.

**Comment by baiyancheng20** [comment]
@liuwei16 @VideoObjectSearch 
Could you update the original code for the following functions?
from .utils.cython_bbox import bbox_overlaps
from .utils.bbox import box_op
The compiled code didn't work because different version of tensorflow.

Also, could you give a detailed description on how to train models? I run many times but also couldn't reproduce as good results as you did.

**Comment by rintiunse** [comment]
@VideoObjectSearch im sure that i run the code on GPU(1080TI). But it still  spent 2s per image(CityPersons).

**Comment by wsz1029** [comment]
> @Chen94yue Sorry for delayed reply. Did you decrease the learning rate and load the trained weights to continue training?

I have a problem that what  the end total loss value should be?

**Comment by Chen94yue** [comment]
> > @Chen94yue Sorry for delayed reply. Did you decrease the learning rate and load the trained weights to continue training?
> 
> I have a problem that what the end total loss value should be?
I haven't finished training yet. I'm not sure.
